pkg/controller/cluster_controller.go

```diff
@@ -38,9 +38,10 @@ import (
 // ClusterReconciler reconciles a CrdbCluster object
 type ClusterReconciler struct {
 	client.Client
-	Log     logr.Logger
-	Scheme  *runtime.Scheme
-	Actions []actor.Actor
+	Log      logr.Logger
+	Scheme   *runtime.Scheme
+	Actors   map[api.ActionType]actor.Actor
```

Comment:
This is what I was talking about above. If Director knew about the actors map, the reconciler wouldn't have to.